<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QMainWindow" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="SnakePage">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="snakeBtn">
        <property name="text">
         <string>贪吃蛇历史记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaSnake">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>222</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="TetrisPage">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="tetrisBtn">
        <property name="text">
         <string>俄罗斯方块历史记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaTetris">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>222</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="MinesPage">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="minesBtn">
        <property name="text">
         <string>扫雷历史记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollAreaMines">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>521</width>
           <height>222</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="meneFile">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <addaction name="meneFile"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
